### DISCUSSION TITLE
BUILD FAILED (macOS 11.5.1 using ruby-build 20211019)

### DISCUSSION THREAD

**Discussion by NitinChronext** [rbenv/ruby-build#1867] [Build failures]
**rbenv install 3.0.2** or 3.0.0 fails on **macOS 11.5.1** with below issues.

`Downloading openssl-1.1.1l.tar.gz...
-> https://dqw8nmjcqpjn7.cloudfront.net/0b7a3e5e59c34827fe0c3a74b7ec8baef302b98fa80088d7f9153aa16fa76bd1
Installing openssl-1.1.1l...

BUILD FAILED (macOS 11.5.1 using ruby-build 20211019)

Inspect or clean up the working tree at /var/folders/5b/_n07b16j4t1104g4zn4j8rd40000gn/T/ruby-build.20211028162821.69642.evzpTP
Results logged to /var/folders/5b/_n07b16j4t1104g4zn4j8rd40000gn/T/ruby-build.20211028162821.69642.log

Last 10 log lines:
20 errors generated.
make[1]: *** [apps/opt.o] Error 1
make[1]: *** [apps/app_rand.o] Error 1
20 errors generated.
20 errors generated.
make[1]: *** [apps/s_cb.o] Error 1
make[1]: *** [apps/s_socket.o] Error 1
20 errors generated.
make[1]: *** [apps/apps.o] Error 1
make: *** [all] Error 2
`

**Comment by brrusselburg** [comment]
FWIW, I also got this error while trying to install 2.6.7

**Comment by muhammadshake** [comment]
getting the same error. What could be the issue? Any fix for this?

**Comment by sgraciamyriad** [comment]
I don't want to assume anything...
but do you all utilize new macbooks with M1 chips?
